content/en/agent/guide/operator-advanced.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Advanced setup for Datadog Operator
+kind: faq
+further_reading:
+  - link: 'agent/kubernetes/log'
+    tag: 'Documentation'
+    text: 'Datadog and Kubernetes'
+---
+
+[The Datadog Operator][1] is in public beta. The Datadog Operator is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
+
+## Prerequisites
+
+Using the Datadog Operator requires the following prerequisites:
+
+- **Kubernetes Cluster version >= v1.14.X**: Tests were done on versions >= `1.14.0`. Still, it should work on versions `>= v1.11.0`. For earlier versions, because of limited CRD support, the operator may not work as expected.
+- [`Helm`][2] for deploying the `datadog-operator`.
+- [`Kubectl` CLI][3] for installing the `datadog-agent`.
+
+## Deploy the Datadog Operator
+
+To use the Datadog Operator, deploy it in your Kubernetes cluster. Then create a `DatadogAgent` Kubernetes resource that contains the Datadog deployment configuration:
+
+1. Download the [Datadog Operator project zip ball][4]. Source code can be found at [`DataDog/datadog-operator`][1].
```

Comment:
@ceciliawatt the operator is now available on Helm, so we should replace #1 and #2 steps with:

1) helm repo add datadog https://helm.datadoghq.com
2) helm install datadog/datadog-operator --version 0.1.0

Reply:
(from https://hub.helm.sh/charts/datadog/datadog-operator)

Reply:
the version is not mandatory. if they want to always install the latest. 
So I think it is better to not document: `--version 0.1.0`